File: 2019/1/1-1.py
#!usr/bin/python

import math

def fuel_needed(mass):
    return math.floor(mass/3) - 2

answer = 0

with open('input.txt') as f:
    for line in f.readlines():
        answer += fuel_needed(int(line.strip('\n')))

print('%s\n' % (answer, ))

File: 2015/1/1-1.go
package main

import (
    "fmt"
)

func main() {
    instructions := getFileLines("input.txt")[0]
    floor := 0

    for _, c := range instructions {
        if c == '(' {
            floor += 1
        }
        if c == ')' {
            floor -= 1
        }
    }

    fmt.Printf("Floor %v\n", floor)
}
